core/lib/mini_merkle_tree/benches/tree.rs

```diff
@@ -10,11 +10,7 @@ const TREE_SIZES: &[usize] = &[32, 64, 128, 256, 512, 1_024];
 fn compute_merkle_root(bencher: &mut Bencher<'_>, tree_size: usize) {
     let leaves = (0..tree_size).map(|i| [i as u8; 88]);
     let tree = MiniMerkleTree::new(leaves, None);
-    bencher.iter_batched(
-        || tree.clone(),
-        MiniMerkleTree::merkle_root,
-        BatchSize::SmallInput,
-    );
+    bencher.iter_batched(|| &tree, MiniMerkleTree::merkle_root, BatchSize::SmallInput);
```

Comment:
Since `merkle_root()` (and `merkle_root_and_path()` below) no longer consume the tree, you can just use `iter()` here and below:

```rust
bencher.iter(|| tree.merkle_root());
```